opa-manager-operator/docs/policy-spec-example.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
Seems unfortunate to have this example duplicated between the charms. Maybe remove it from one of them?

Reply:
Please remove duplicated examples.

Reply:
removed on audit